### PR TITLE
Network working group notes, September 7th 2023

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

This pull request adds notes from Network Working group meeting on September 7th, 2023.

[Rendered](https://github.com/tock/tock/blob/53505a5926608fc896b05e2d781538bb00dc6cf8/doc/wg/network/notes/network-notes-2023-09-07.md)

### Testing Strategy

This pull request was tested by skimming it after starting the PR, then realizing that I'd missed a TODO, then amending my commit. 

### TODO or Help Wanted

Should be good to go.

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Review by lschuermann — APPROVED** [review]

File: doc/wg/network/notes/network-notes-2023-09-07.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Tock Network WG Meeting Notes
+
+###### tags: `Meeting` `Network`
+
+2023-09-07
+===
+
+:::info
```
Comment (alexandruradovici) [inline doc/wg/network/notes/network-notes-2023-09-07.md:8]:
```suggestion
```

File: doc/wg/network/notes/network-notes-2023-09-07.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    1. Updates
+    2. Buffer Management
+        [name=Leon]
+    3. Discuss 14.5 Layer Security (maybe during updates?)
+        [name=Tyler]
+- **References:**
+    - [14.5 Layer Security](https://github.com/tock/tock/pull/3652)
+    - 
+
+:::
```
Comment (alexandruradovici) [inline doc/wg/network/notes/network-notes-2023-09-07.md:28]:
```suggestion
```

File: doc/wg/network/notes/network-notes-2023-09-07.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    - Tyler Potyondy
+    - Branden Ghena
+    - Leon Schuermann
+    - Cristian Rusu
+    - Felix Mada
+    - Ioan-Cristian Cirstea
+- **Agenda**
+    1. Updates
+    2. Buffer Management
+        [name=Leon]
```
Comment (alexandruradovici) [inline doc/wg/network/notes/network-notes-2023-09-07.md:21]:
```suggestion
```

File: doc/wg/network/notes/network-notes-2023-09-07.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    3. Discuss 14.5 Layer Security (maybe during updates?)
+        [name=Tyler]
```
Comment (alexandruradovici) [inline doc/wg/network/notes/network-notes-2023-09-07.md:23]:
```suggestion
```

File: doc/wg/network/notes/network-notes-2023-09-07.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# Tock Network WG Meeting Notes
+
+###### tags: `Meeting` `Network`
```
Comment (alexandruradovici) [inline doc/wg/network/notes/network-notes-2023-09-07.md:3]:
```suggestion
```

**Review by alexandruradovici — COMMENTED** [review]
I think some of the markup language from Hack.md is not valid for GitHub.

**Pull request merged**